tests_tensorflow/resources/buyer/buyer.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs')
+
+const modelsArtifacts = [
+  {
+    modelTopology: require('./model0/modelTopology.json'),
+    weightSpecs: require('./model0/weightSpecs.json'),
+    weightData: require('./model0/weightData'),
+  },
+  {
+    modelTopology: require('./model1/modelTopology.json'),
+    weightSpecs: require('./model1/weightSpecs.json'),
+    weightData: require('./model1/weightData'),
+  },
+  {
+    modelTopology: require('./model2/modelTopology.json'),
+    weightSpecs: require('./model2/weightSpecs.json'),
+    weightData: require('./model2/weightData'),
+  },
+  {
+    modelTopology: require('./model3/modelTopology.json'),
+    weightSpecs: require('./model3/weightSpecs.json'),
+    weightData: require('./model3/weightData'),
+  },
+  {
+    modelTopology: require('./model4/modelTopology.json'),
+    weightSpecs: require('./model4/weightSpecs.json'),
+    weightData: require('./model4/weightData'),
+  },
+];
+
+const tf = require('@tensorflow/tfjs')
+
+generateBid = async function (interestGroup, auctionSignals, perBuyerSignals, trustedBiddingSignals, browserSignals) {
```

Comment:
It would be nice to measure which part is the most heavy here (loading layers, doing calculations etc.). Just for debug purposes you probably could use console.log at the end of generateBid().